runtime/src/bank_utils.rs

```diff
@@ -7,9 +7,12 @@ use {
     solana_sdk::{pubkey::Pubkey, signature::Signer},
 };
 use {
-    solana_sdk::transaction::SanitizedTransaction,
+    log::{debug, log_enabled, trace},
+    solana_program_runtime::loaded_programs::Stats,
```

Comment:
Since the runtime is now importing the `Stats` struct, can we rename it to `LoadedProgramStats`?